.coveralls.github-actions.yml

```diff
@@ -0,0 +1,3 @@
+service_name: github-actions
```

Comment:
Can we pass these as direct parameter to the coveralls as parameters?
```
./vendor/bin/php-coveralls -v --coverage_clover='./dist/tests.clover' --json_path='./dist/coveralls-upload.json'
```
This will allows us to have fewer random files in the repo root.